node/cmd/guardiand/node.go

```diff
@@ -1658,12 +1678,18 @@ func runNode(cmd *cobra.Command, args []string) {
 	}
 
 	if shouldStart(aptosRPC) {
+		// Determine if we should use indexer mode (only aptosIndexerRPC required)
+		useIndexer := *aptosIndexerRPC != ""
+
 		wc := &aptos.WatcherConfig{
-			NetworkID: "aptos",
-			ChainID:   vaa.ChainIDAptos,
-			Rpc:       *aptosRPC,
-			Account:   *aptosAccount,
-			Handle:    *aptosHandle,
+			NetworkID:    "aptos",
+			ChainID:      vaa.ChainIDAptos,
+			Rpc:          *aptosRPC,
+			Account:      *aptosAccount,
+			Handle:       *aptosHandle,
+			IndexerRpc:   *aptosIndexerRPC,
+			IndexerToken: *aptosIndexerToken,
```

Comment:
Should there be a check here that the indexer token is non-empty when indexer mode is selected?

Reply:
We are allowing for the possibility that future indexers may be run by the guardians themselves and then won't need a token.